### PR TITLE
Clean Up Unused Imports

### PR DESCRIPTION
**Pull request by knazir**
Ran `optimize imports` from IntelliJ on the entire top-level package.

### CODE REVIEW

**Comment by bdmogal** [comment]
the import changes are fine. But it looks like your IDE made a fair bit of unintended indentation changes as well. Can you revert them?

**Comment by knazir** [comment]
@bdmogal should be fixed now

File: cdap-ui/app/cdap/styles/img/cask_footer_logo.svg [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-			 M0,379.9v94c26.4,25.6,68,25.3,94.4,0v-29.4H72.8v18.6c-11.5,11.5-40.2,11.1-51.3,0v-72.2c11.6-11.6,39.6-11.4,51.3,0v18.5h21.6
-			v-29.5C68,354.6,26.4,354.6,0,379.9z M345.9,416.2c-8.3-9.2-8-25.6,0-34h48.8l11.4-21.3h-72.2c-20.6,20.6-20.9,56.2-0.3,76.6h50.8
-			c8.3,9.3,8,25.6-0.1,34h-51.8l-11.4,21.3h75.3c20.6-20.6,20.9-56.2,0.3-76.7H345.9z M159.2,379.9V493h21.6v-33.9H232V493l21.6,0
-			V379.8C227.1,354.6,185.6,354.6,159.2,379.9z M232,437.6h-51.3l0.1-46.7c12.5-12.3,40.5-10.6,51.3,0V437.6z M390.5,42.1
-			c-58.7-56.2-151.2-56.2-210,0v182.4c58.7,57,151.2,56.2,210,0V42.1z M357.5,106.6h-33V78.5c-17.7-17.4-60.4-17.7-78.1,0v110.1
-			c16.8,16.9,60.6,17.6,78.1,0v-28.3h33V205c-40.3,38.5-103.7,39.1-144,0V61.6c40.3-38.5,103.7-38.5,144,0V106.6z"/>
-	</g>
-</g>
-</svg>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (bdmogal) [inline cdap-ui/app/cdap/styles/img/cask_footer_logo.svg]: this shouldn't be changed

File: cdap-ui/app/styles/img/cask_footer_logo.svg [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-			 M0,379.9v94c26.4,25.6,68,25.3,94.4,0v-29.4H72.8v18.6c-11.5,11.5-40.2,11.1-51.3,0v-72.2c11.6-11.6,39.6-11.4,51.3,0v18.5h21.6
-			v-29.5C68,354.6,26.4,354.6,0,379.9z M345.9,416.2c-8.3-9.2-8-25.6,0-34h48.8l11.4-21.3h-72.2c-20.6,20.6-20.9,56.2-0.3,76.6h50.8
-			c8.3,9.3,8,25.6-0.1,34h-51.8l-11.4,21.3h75.3c20.6-20.6,20.9-56.2,0.3-76.7H345.9z M159.2,379.9V493h21.6v-33.9H232V493l21.6,0
-			V379.8C227.1,354.6,185.6,354.6,159.2,379.9z M232,437.6h-51.3l0.1-46.7c12.5-12.3,40.5-10.6,51.3,0V437.6z M390.5,42.1
-			c-58.7-56.2-151.2-56.2-210,0v182.4c58.7,57,151.2,56.2,210,0V42.1z M357.5,106.6h-33V78.5c-17.7-17.4-60.4-17.7-78.1,0v110.1
-			c16.8,16.9,60.6,17.6,78.1,0v-28.3h33V205c-40.3,38.5-103.7,39.1-144,0V61.6c40.3-38.5,103.7-38.5,144,0V106.6z"/>
-	</g>
-</g>
-</svg>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (bdmogal) [inline cdap-ui/app/styles/img/cask_footer_logo.svg]: shouldn't be changed?

File: cdap-docs/hydrator-manual/source/_images/etl-pipeline.svg [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
....765625 -8.921875q0 -1.734375 -0.390625 -3.453125q-0.328125 -1.390625 -0.890625 -2.671875q-0.359375 -0.84375 -1.484375 -2.78125l1.1875 0q1.75 2.328125 2.578125 4.671875q0.71875 2.015625 0.71875 4.234375q0 2.5 -0.96875 4.84375q-0.953125 2.328125 -2.328125 4.078125z" fill-rule="nonzero"></path><path fill="#000000" fill-opacity="0.0" d="m163.57217 78.80315l26.51332 0l0 0.06299591l26.526047 0" fill-rule="nonzero"></path><path stroke="#000000" stroke-width="2.0" stroke-linejoin="round" stroke-linecap="butt" d="m163.57217 78.80315l26.513306 0l0 0.06299591l14.526062 0" fill-rule="evenodd"></path><path fill="#000000" stroke="#000000" stroke-width="2.0" stroke-linecap="butt" d="m204.61154 82.16961l9.076202 -3.3034668l-9.076202 -3.3034668z" fill-rule="evenodd"></path><path fill="#000000" fill-opacity="0.0" d="m411.937 78.80315l53.039368 0" fill-rule="nonzero"></path><path stroke="#000000" stroke-width="2.0" stroke-linejoin="round" stroke-linecap="butt" d="m411.937 78.80315l41.039368 0" fill-rule="evenodd"></path><path fill="#000000" stroke="#000000" stroke-width="2.0" stroke-linecap="butt" d="m452.97638 82.10661l9.076202 -3.3034668l-9.076202 -3.3034592z" fill-rule="evenodd"></path></g></svg>
```
Comment (bdmogal) [inline cdap-docs/hydrator-manual/source/_images/etl-pipeline.svg]: shouldn't be changed?

File: cdap-archetypes/cdap-spark-scala-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml [hunk 4]
```diff
@@ -14,9 +14,8 @@
   License for the specific language governing permissions and limitations under
   the License.
   -->
-<archetype-descriptor xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0 http://maven.apache.org/xsd/archetype-descriptor-1.0.0.xsd"
-  name="cdap-spark-scala-archetype">
+<archetype-descriptor xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype/1.0.0"
```
Comment (bdmogal) [inline cdap-archetypes/cdap-spark-scala-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml]: shouldn't be changed

File: cdap-archetypes/cdap-app-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml [hunk 5]
```diff
@@ -14,9 +14,8 @@
   License for the specific language governing permissions and limitations under
   the License.
   -->
-<archetype-descriptor xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0 http://maven.apache.org/xsd/archetype-descriptor-1.0.0.xsd"
-  name="cdap-app-archetype">
+<archetype-descriptor xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype/1.0.0"
```
Comment (bdmogal) [inline cdap-archetypes/cdap-app-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml]: shouldn't be changed

**Comment by knazir** [comment]
@bdmogal Removed all changes to `xml` and `svg` files.

File: cdap-docs/hydrator-manual/source/_images/etl-pipeline.svg [hunk 6]
```diff
@@ -1,4 +1,3 @@
...
-
```
Comment (bdmogal) [inline cdap-docs/hydrator-manual/source/_images/etl-pipeline.svg:4]: revert this change

File: cdap-docs/_common/common_conf.py [hunk 7]
```diff
@@ -38,12 +38,13 @@
...
 ###############################################################
 
-import sys
 import os
 import os.path
 import subprocess
+import sys
 from datetime import datetime
 
+
```
Comment (bdmogal) [inline cdap-docs/_common/common_conf.py]: remove empty line

**Comment by bdmogal** [comment]
lgtm, as long as build succeeds locally. Just to be sure, since a lot of python files have been modified, please run the docs quick build on this branch before merging - http://builds.cask.co/browse/CDAP-DQB

**Comment by knazir** [comment]
Guild is breen: http://builds.cask.co/browse/CDAP-DUT4802-3 going to squash and merge.
